blue-note/templates/home.html

```diff
@@ -0,0 +1,57 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"left"}} -->
+<main class="wp-block-group">
+
+	<!-- wp:heading -->
+	<h2 class="wp-block-heading">Latest Posts</h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:query {"queryId":0,"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"displayLayout":{"type":"flex","columns":3},"align":"full","layout":{"type":"default"}} -->
+	<div class="wp-block-query alignfull">
+		<!-- wp:post-template -->
+			<!-- wp:post-featured-image /-->
+			
+			<!-- wp:post-title {"isLink":true} /-->
+			
+			<!-- wp:post-date /-->
+		<!-- /wp:post-template -->
+	
+		<!-- wp:query-no-results -->
+		<!-- wp:paragraph {"placeholder":"Add text or blocks that will display when a query returns no results."} -->
+		<p></p>
+		<!-- /wp:paragraph -->
+		<!-- /wp:query-no-results -->
+	</div>
+	<!-- /wp:query -->
+
+	<!-- wp:spacer -->
+	<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+
+	<!-- wp:heading -->
+	<h2 class="wp-block-heading">By Topic</h2>
+	<!-- /wp:heading -->
+
+	<!-- wp:query {"queryId":0,"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"displayLayout":{"type":"flex","columns":3},"align":"full","layout":{"type":"default"}} -->
```

Comment:
These are not really posts, but categories. How should we show these?

Reply:
I think the archive title and the archive description only work on the archive template.
Maybe it is an advanced variation of a category block? Untested.

Reply:
https://github.com/WordPress/gutenberg/issues/49094 I think this is what we want